experimental/dds/tree2/src/feature-libraries/forestRepairDataStore.ts

```diff
@@ -74,8 +87,10 @@ export class ForestRepairDataStore implements RepairDataStore {
 					switch (type) {
 						case Delta.MarkType.MoveOut:
 						case Delta.MarkType.Delete: {
+							cursor.enterNode(index);
```

Comment:
is this change a bug fix? It should be noted in the PR description (and/or title).

Reply:
Bug fix. Good point.